proposals/Level-1.md

```diff
@@ -313,36 +300,38 @@ This section describes change in the
 
 The `event index space` indexes all imported and internally-defined events,
 assigning monotonically-increasing indices based on the order defined in the
-import and event sections. Thus, the index space starts at zero with imported
-events followed by internally-defined events in the [event
+import and event sections. Thus, the index space starts at zero with
+imported events, followed by internally-defined events in the [event
 section](#event-section).
 
-The event index space defines the (module) static version of runtine event
-tags. These (runtime) event tags are guaranteed to be unique over all loaded
+The event index space defines the (module) static version of runtine
+event tags. For event indicies that are not imported/exported, the
+corresponding event tag is guaranteed to be unique over all loaded
 modules.
 
-For event indices imported/exported, unique event tags are created for
-each unique name imported/exported, and are aliased to the corresponding
-event index defined in each module.
+For event indices imported/exported, unique event tags are created for each
+unique name imported/exported, and are aliased to the corresponding event index
+defined in each module.
 
 ## Changes to the binary model
 
-This section describes changes in the [binary encoding design
-document](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).
+This section describes changes in
+the
+[binary encoding design document](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md).
 
 
 ### Data Types
 
 #### except_ref
 
-An exception reference points to an exception. The size is fixed, but unknown in
-WebAssembly.
+An exception reference points to an exception. The size is defined by
+the embedder.
```

Comment:
I would drop this -- why require that there is a fixed size? That's not observable. In particular, I would avoid formulations that suggest that the semantics depends the an embedder.

Reply:
Well, the specification does not enforce specific implementation strategies. A hypothetical engine could heap-allocate everything, eschew the stack entirely for CPS, and still be conforming.